Sources/Datadog/RUMMonitor.swift

```diff
@@ -56,6 +56,8 @@ public class RUMMonitor: DDRUMMonitor {
     internal let contextProvider: RUMCurrentContext
     /// Time provider.
     private let dateProvider: DateProvider
+    /// Attributes associated with every command.
+    private var RUMAttributes: [AttributeKey: AttributeValue] = [:]
```

Comment:
Convention: variable names should start with a lowercase
```suggestion
    private var rumAttributes: [AttributeKey: AttributeValue] = [:]
```
or just `attributes` as we are in `RUM` monitor.